lib/rouge/lexers/groovy.rb

```diff
@@ -7,7 +7,7 @@ class Groovy < RegexLexer
       title "Groovy"
       desc 'The Groovy programming language (http://www.groovy-lang.org/)'
       tag 'groovy'
-      filenames '*.groovy', 'Jenkinsfile'
+      filenames '*.groovy', 'Jenkinsfile', '*.Jenkinsfile'
```

Comment:
**suggestion:** Could you add a spec for this new file name, similar to [this](https://github.com/rouge-ruby/rouge/blob/master/spec/lexers/groovy_spec.rb#L11).

Reply:
Not that familiar with that syntax. Something like this should work, right?

```ruby
    it 'guesses by filename' do
      assert_guess :filename => 'foo.groovy'
      assert_guess :filename => 'Jenkinsfile'
      assert_guess :filename => 'foo.Jenkinsfile'
    end
```

Reply:
@danilaml Yep that will do 👍🏼